src/assets/i18n/cs.json5

```diff
@@ -2624,6 +2624,8 @@
   "item.page.link.full": "Zobrazit celý záznam",
   // "item.page.link.simple": "Simple item page",
   "item.page.link.simple": "Zobrazit minimální záznam",
+  // "item.page.users.note": "User's note:",
+  "item.page.users.note": "Vzkaz od předkladatele:",
```

Comment:
This is certainly wrong translation. It will be unclear, if english one is used. Can we change it? perhaps to "submitter" or something such.

Reply:
Předkladatel is czech translation of the submitter

Reply:
yes, then why in the english translation, there is "User"

Reply:
Good point, the v5 has a message 'User's note' without any translation. I translated it as submitter because it makes more sense for me.

Reply:
I think it should be kept the same. Better to both change to submitter, but if no, keep User in czech translation as well.

Reply:
1. I do not want to change v5 messages.
2. I want to keep it translated somehow
3. So the right translation should be "Vzkaz od uživatele"?

Reply:
Yes, that's what I think.

Reply:
But I think the "předkladatel" is more specific translation to give the information from where is that note.

Reply:
It is better. But it is not a translation of the english version. I think that should not be allowed. That's why I am against this. I will not approve it like this.

Reply:
You won, changed. Please resolve this conversation.